src/cards/index.tsx

```diff
@@ -151,8 +151,8 @@ const Cards = React.forwardRef(function <T = any>(
               visibleSections={visibleSections}
               updateShiftToggle={updateShiftToggle}
               onFocus={onCardFocus}
-              ariaDescribedby={cardsHeaderId}
```

Comment:
removed `aria-describedby` entirely as it was duplicating `aria-labelledby`, and we don't provide descriptions on similar components, e.g. table